prow/plugins/trigger/generic-comment.go

```diff
@@ -74,7 +74,7 @@ func handleGenericComment(c Client, trigger plugins.Trigger, gc github.GenericCo
 	}
 
 	// Skip untrusted users comments.
-	trusted, err := TrustedUser(c.GitHubClient, trigger, commentAuthor, org, repo)
+	trusted, err := plugins.TrustedUser(c.GitHubClient, trigger.OnlyOrgMembers, trigger.TrustedOrg, commentAuthor, org, repo)
```

Comment:
This was already exported, why did it need to move?

Reply:
@stevekuznetsov The reason why I moved it is because it was in the `trigger` package before. As it's used by both `trigger` and `dco` packages now, I thought it should be placed at the higher level. I'm open to suggestions should I move it back or move to some other place.

Reply:
I think we can just keep it in place

/shrug

Reply:
@stevekuznetsov Just to make sure I understood correctly. Keep it in the place as in return it to the `trigger` package or keep it in the current place (`prow/plugins/trusteduser.go`)? :sweat_smile:

Reply:
Keep it in `trigger` -- will make these changes smaller

Reply:
I moved it back to `trigger`. However, I kept the change which made it non-dependent on the `Trigger` structure, which required me to do this https://github.com/kubernetes/test-infra/pull/11832/commits/bd45e88813a015b2cefc7e98f341738341d61082? WDYT about this?